homeassistant/components/tasmota/light.py

```diff
@@ -95,7 +94,7 @@ class TasmotaLight(
     def __init__(self, **kwds: Any) -> None:
         """Initialize Tasmota light."""
         self._supported_color_modes: set[str] | None = None
-        self._supported_features = 0
+        self._attr_supported_features = 0
```

Comment:
```suggestion
```

Reply:
This is the default already and thus can be removed?

Reply:
Yes, I removed it, and then reverted it because I got mixed up and thought I'd made a mistake.
I've removed it again :)